Assets/MixedRealityToolkit.SDK/Experimental/Inspectors/TapToPlaceInspector.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Experimental.Utilities;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.Utilities.Solvers;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Inspectors
+{
+    [CustomEditor(typeof(TapToPlace))]
+    internal class TapToPlaceInspector : UnityEditor.Editor
```

Comment:
Nit: Also comments for this class as well.

Reply:
Comment added